src/services/getActiveMachines.ts

```diff
@@ -0,0 +1,5 @@
+import { getBackendBaseUrl } from "./getBackendBaseUrl";
```

Comment:
should be a part of `utils/constants.ts` or a `.env` file. I'd opt for the latter (guess why?)